addons/mail/static/src/models/message_action.js

```diff
@@ -0,0 +1,139 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, insertAndReplace, replace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'MessageAction',
+    identifyingFields: [[
+        'messageActionListOwnerAsDelete',
+        'messageActionListOwnerAsEdit',
+        'messageActionListOwnerAsMarkAsRead',
+        'messageActionListOwnerAsReaction',
+        'messageActionListOwnerAsReplyTo',
+        'messageActionListOwnerAsToggleCompact',
+        'messageActionListOwnerAsToggleStar',
+    ]],
+    recordMethods: {
```

Comment:
Again, rebase master and adapt with new changes in master.
Here `identifyingMode: 'xor'` and each field listed here has `identifying: true`